backend/route/router.js

```diff
@@ -1,11 +1,20 @@
 const express = require('express');
 const router = express.Router();
-
+const events = require('../events.json')
 router.get("/home", (req, res) => {
     res.send("it is working");
 })
 
+router.get("/all-events", (req, res) => {
```

Comment:
For our purposes, I think this is fine, but there is a convention around route naming. I would name them slightly differently here. Something like:

`GET /events` would return all the events
`GET /events/:id` would return a single event
`POST /events` would create an event

Reply:
This is generally called the "RESTful" convention btw